js/main.js

```diff
@@ -19,4 +19,76 @@ $form.addEventListener('submit', function (event) {
   data.entries.unshift(formData);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+  var $ul = document.querySelector('ul');
+  $ul.prepend(renderEntry(formData));
+  $dataViewEntryForm.className = 'container data-form hidden';
+  $dataViewEntries.className = 'container data-entries';
+  var $entriesDataView = $dataViewEntries.getAttribute('data-view');
+  localStorage.setItem('code-journal-data-view', $entriesDataView);
+  var $noEntriesMessage = document.querySelector('.no-entries');
+  $noEntriesMessage.className = 'no-entries hidden';
 });
+
+function renderEntry(entry) {
+  var $li = document.createElement('li');
+  $li.setAttribute('class', 'row entry');
+  var $img = document.createElement('img');
+  $img.setAttribute('src', entry.photoURL);
+  $img.setAttribute('alt', entry.title);
+  $img.setAttribute('class', 'column-half img-half');
+  $li.appendChild($img);
+  var $div = document.createElement('div');
+  $div.setAttribute('class', 'column-half');
+  $li.appendChild($div);
+  var $h3 = document.createElement('h3');
+  var $title = document.createTextNode(entry.title);
+  $h3.appendChild($title);
+  $div.appendChild($h3);
+  var $p = document.createElement('p');
+  var $notes = document.createTextNode(entry.notes);
+  $p.appendChild($notes);
+  $div.appendChild($p);
+  return $li;
+}
+
+window.addEventListener('DOMContentLoaded', function (event) {
+  var $ul = document.querySelector('ul');
+  for (var i = 0; i < data.entries.length; i++) {
+    $ul.appendChild(renderEntry(data.entries[i]));
+  }
+  var JSONDataView = localStorage.getItem('code-journal-data-view');
+  if (JSONDataView === 'entries') {
+    $dataViewEntryForm.className = 'container data-form hidden';
+    $dataViewEntries.className = 'container data-entries';
+  } else {
+    $dataViewEntryForm.className = 'container data-form';
+    $dataViewEntries.className = 'container data-entries hidden';
+  }
```

Comment:
This is a very manual way of handling view swapping, I suggest you try to implement a system more similar to what you did in the `javascript-view-swapping` exercise that used a loop over all view containers.